workalendar/tests/test_asia.py

```diff
@@ -788,3 +788,33 @@ def test_is_holiday_performance(self):
             self.cal.is_holiday(random_date)
         israel_time = time.time() - timer
         self.assertGreater(japan_time * 3, israel_time)
+
+
+class Philippines(GenericCalendarTest):
+
+    cal_class = Philippines
+
+    def test_year_2021(self):
```

Comment:
That's very cool to have a test class. Would it be possible to have another year tested? 2020, for example?

Reply:
Added year 2020 test

Reply:
👍